addons/hr_timesheet/static/src/js/timesheet_uom.js

```diff
@@ -153,6 +153,7 @@ if (widgetName === 'float_toggle') {
 }
 
 fieldRegistry.add('timesheet_uom', FieldTimesheetUom);
+fieldRegistry.add('float_factor', FieldTimesheetFactor);
```

Comment:
This overwrites the `float_factor` field defined in the web module :/
Not sure it is intended